cartridges/int_algolia/cartridge/scripts/algolia/steps/algoliaProductIndex.js

```diff
@@ -375,7 +379,9 @@ exports.afterStep = function(success, parameters, stepExecution) {
     // "success" conveys whether an error occurred in any previous chunks or not.
     // Any prior return statements will set success to false (even if it returns Status.OK).
 
-    products.close();
+    if (products.close) {
```

Comment:
Do we need that?

Reply:
Yes, because the error about locales not enabled ([here](https://github.com/algolia/algoliasearch-sfcc-b2c/pull/176/files#diff-c370b2bd57ec407c9d71be2fb58978c3b450b6ed92897566026890f1d96c9707R181)) is thrown before the call to `ProductMgr.queryAllSiteProducts()`
In that case, `products` is a simple empty array, so trying to call `close()` on it will throw an exception